store/rules/preLoadThreads.js

```diff
@@ -1,38 +1,40 @@
 "use strict";
 
-var core, config, store;
+var core, store;
 //var entityOps = require("./../entity-ops.js");
 
 var query;
 
-module.exports = function(c, conf, s) {
-	core = c;
-	config = conf;
-	store = s;
+module.exports = function() {
+	core = arguments[0];
+	store = arguments[2];
 
-	query = require("../bulkQuery.js")(core, store, 'threads');
+	query = require("../bulkQuery.js")(core, store, "threads");
 
 	core.on("setstate", function(changes, next) {
 		var user = store.get("user") || "",
 			room = store.get("nav", room) || "",
-			regex = new RegExp("_" + user + "$");
+			regex = new RegExp("_" + user + "$"),
```

Comment:
For completeness, `regex = new RegExp("_" + RegExp.escape(user) + "$")`

We might have to add RegExp.escape to polyfills.